snapcraft/plugins/make.py

```diff
@@ -32,6 +32,10 @@
     - make-parameters:
       (list of strings)
       Pass the given parameters to the make command.
+
+    - make-install-var:
+      (string; default: DESTDIR)
+      Use the variable as the install target.
```

Comment:
This is a little confusing-- "install" **is** the target. Perhaps something like "Use this variable to redirect the installation into the snap" or something like that.